stdlib/private/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ endif()
 if(SWIFT_BUILD_SDK_OVERLAY)
   # SwiftPrivateThreadExtras makes use of Darwin/Glibc, which is part of the
   # SDK overlay. It can't be built separately from the SDK overlay.
+  add_subdirectory(DifferentiationUnittest)
```

Comment:
Oh, does this need `if(SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING)`? I'll add that here and see what happens.